.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,10 +15,24 @@ is merged.
 See: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
 -->
 
-
 #### What does this implement/fix? Explain your changes.
 
 
+#### Checklist
+
+<details>
+  <summary>
+   Code change checklist...
+  </summary>
+ 
```

Comment:
Is the PR labelled correctly as feature or bug so it appears in the autogenerated release notes?